perception/tensorrt_yolox/config/yolox_s_plus_opt.yaml

```diff
@@ -0,0 +1,15 @@
+/**:
```

Comment:
Please rename the file name to param.yaml

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/6239/commits/d88251c03f73f4c597850b6225d5e0efc23db8a3